c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -1435,7 +1460,12 @@ class DeviceCachingAllocator {
           stats.inactive_split_bytes[stat_type],
           net_change_inactive_split_size);
       update_stat(stats.active[stat_type], -1);
-      update_stat(stats.active_bytes[stat_type], -original_block_size);
+      update_stat(
+          stats.active_bytes[stat_type],
+          static_cast<std::int64_t>(-original_block_size));
```

Comment:
Should these negations of unsigned quantities happen after the conversion to int64_t.

Reply:
Addressed